modules/storages/app/common/storages/peripherals/one_drive_registry.rb

```diff
@@ -58,6 +58,11 @@ module Peripherals
       namespace("models") do
         register(:managed_folder_identifier, ManagedFolderIdentifier::OneDrive)
       end
+
+      namespace("authentication") do
+        register(:userless, StorageInteraction::AuthenticationStrategies::OneDriveStrategies::UserLess)
```

Comment:
we still have the chance now to name this `user_less`. At least my spell checker in the IDE marks that xD

Reply:
WDYT of `application` and `user`.

Reply:
I do not have a strong take on that. I was kind of hyped by the words `user-less` and `user-bound`, but there is no deeper reasoning behind.

I'd be fine with `application` and `user`, too.